cli/UploadSoFiles.ts

```diff
@@ -0,0 +1,67 @@
+import axios from 'axios';
+import { Command, Option } from 'commander';
+import FormData from 'form-data';
+import fs from 'fs';
+
+interface UploadSoFilesOptions {
+  arch: 'x86' | 'x86_64' | 'arm64-v8a' | 'armeabi-v7a';
+  file: string;
+  token: string;
+  name: string;
+  api_key: string;
+}
+
+export const UploadSoFilesCommand = new Command();
```

Comment:
How about we add a comment to explain the script like the below, with sample usage if needed:
```
/**
 * This script uploads .so files to the specified endpoint used in NDK crash reporting.
 * Usage: node upload-so-files.js --arch <arch> --file <path> --api_key <key> --token <token> --name <name>
 */
 ```

The above is but a sample, please make sure to adjust it somehow 🙏